.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.5","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["C:\\A\\SCOOP\\apps\\adoptopenjdk-hotspot\\current/bin/java","-Xms100m","-Xmx100m","-classpath","C:/Users/admctv/AppData/Roaming/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=C:/Users/admctv/AppData/Roaming/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
Please remove this file or add `.bsp` to `.gitignore`. This is autogenerated by newer IntelliJ.